### ISSUE TITLE
Min String check

### ISSUE THREAD

**Issue by hongkongkiwi** [danmilon/assurance#2]
Right now there is a len(max) length and len(min,max) for strings, however I think this plugin should also ship with a plugin to check that the length min, e.g. len(100,0) for minimum 100 length and max anything.

**Comment by danmilon** [comment]
I see what you mean. How about `minLength(min)`, `maxLength(max)` ? PR is welcome.

**Comment by hongkongkiwi** [comment]
Yes, that makes most sense, minLength, maxLength and exactlyLength should be the 3 options, len can be used as is by len(min,max).

I wouldn't mind to help but I am not familiar with GIT (I use HG myself).
